code/modules/webhooks/webhook_roundstart.dm

```diff
@@ -4,13 +4,11 @@
 // Data expects a "url" field pointing to the current hosted server and port to connect on.
 /decl/webhook/roundstart/get_message(var/list/data)
 	. = ..()
-	var/desc = "A new round is starting"
-	if(data && data["url"])
-		desc += " on [data["url"]]"
-	desc += "."
+	var/desc = "Gamemode: [SSticker.master_mode]\n"
+	desc += "Players: [GLOB.player_list.len]"
```

Comment:
No URL?

Reply:
Not sure what people want join game from roundend webhook. I made special hook where you can just click and join.
Btw, think this PR need some discussion how those webhooks must look like.